gridstatus/ercot.py

```diff
@@ -1915,6 +1919,64 @@ def _make_bid_curve(df):
 
         return self.parse_doc(df, verbose=verbose)
 
+    @support_date_range("DAY_START")
+    def get_dam_system_lambda(self, date, end=None, verbose=False):
+        """Get Day-Ahead Market System Lambda
+
+        File is typically published around 12:30 pm for the day ahead
+
+        https://www.ercot.com/mp/data-products/data-product-details?id=NP4-523-CD
+
+        Arguments:
+            date (str, datetime): date to get data for
+            end (str, datetime, optional): end time to get data for. If None,
+                return 1 day of data. Defaults to None.
+            verbose (bool, optional): print verbose output. Defaults to False.
+
+        Returns:
+            pandas.DataFrame: A DataFrame with day-ahead market system lambda data
+        """
+        if date == "latest":
+            # Fetch the latest data published after the start of yesterday.
+            # This is either data published yesterday for today, or data published
+            # today for tomorrow.
+            date = pd.Timestamp.now(tz=self.default_timezone).normalize()
```

Comment:
if you look at definition for `_get_document`, you can pass `date="latest"` and it will return the last publish document